Sketch.java

```diff
@@ -16,21 +16,60 @@ public void settings() {
    * values here i.e background, stroke, fill etc.
    */
   public void setup() {
-    background(210, 255, 173);
+    background(135, 206, 235);
   }
 
   /**
    * Called repeatedly, anything drawn to the screen goes here
    */
   public void draw() {
-	  
-	// sample code, delete this stuff
-    stroke(128);
-    line(150, 25, 270, 350);  
-
-    stroke(255);
-    line(50, 125, 70, 50);  
+	  drawGround(1, 1, 86, 125, 70);
+    drawHouse(1, 1, 203, 65, 84, 150, 75, 0, 172, 140, 24, 135, 206, 235, 0, 0, 128);
```

Comment:
as specified in [level 3 requirements ](https://github.com/SACHSTech/ProcessingTask6-CreatingMethods#level-3), it is expected that the program demonstrates "draw a composite object at various locations locations specified by the method parameters."  So there should be multiple calls to `drawhouse` at varoius locations